code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -1192,3 +1192,12 @@
 		/obj/item/stock_parts/manipulator = /obj/item/stock_parts/manipulator/femto,
 		/obj/item/stock_parts/micro_laser = /obj/item/stock_parts/micro_laser/quadultra,
 		/obj/item/stock_parts/scanning_module = /obj/item/stock_parts/scanning_module/triphasic)
+
+/obj/item/circuitboard/clawgame
```

Comment:
/obj/item/circuitboard/machine/clawgame
dont forget to change this where it's used